### DISCUSSION TITLE
Influence of ocean variables on remnant ice

### DISCUSSION THREAD

**Discussion by EllieBowler** [icenet-ai/icenet#295] [General]
Our Canadian partners using IceNet forecasts for the Foxe Basin polar bear survey had an interesting question on forecasting remnant ice. They said as the ice size reduces it becomes more influenced by wind and ocean variables, and wanted to know how well IceNet captures this.

Currently wind vectors are included, but i understood that ocean variables were removed from some models as it was found they didn't contribute very much to performance. I wonder if this picture changes if we focus on accuracy metrics more related to low sea ice concentration, rather than (e.g.) pan-Arctic RMSE? Interested if anyone has perspectives from the modelling side! E.g. @ampersandmcd @pyatsysh, or @JimCircadian as an IceNet vet!  

Discussed this intially with @bnubald, please add if i misinterpreted anything!

**Comment by JimCircadian** [comment]
I think this highlights the importance of smaller region analysis. The model would be the same, but the influence of those variables might become more pronounced if we evaluate over regions that aren't the entire hemisphere. I will add this to the paper layout @EllieBowler, as it could set a useful precedent, and I will say more soon but it'd be good if people could contribute strategies / approaches to looking at these types of events. The fact that they were kind enough to be specific means we have a direct test case to incorporate... ;)

**Comment by EllieBowler** [comment]
Thanks James! Just noting some more specifics from Amelie here for reference:

> We've run into some broken remnant ice in the Igloolik area. And it seems that depending on the water or winds the amount of remnant broken ice can vary greatly. I was wondering if IceNet forecasts factor in these movements?

> The amount of ice seemed to be a higher percentage than we would've anticipated based on the model results. The area was virtually completely covered in ice between Hecla and Fury strait. We saw another large patch of broken sea ice north of Prince Charles Island as well. That patch was smaller, so we assumed that was diluted across the 25km square resolution, but closer to Igloolik it was a much larger patch. 

They note they have videos and photos of the ice if useful for context. 

Might be interesting to look at the [sea ice probability](https://atmos.washington.edu/~ed/sio_metrics/index.html) for forecasts in that region/time to visualise confidence a bit better. It may also be partly due to limitations of PMR data around archipelagos, which could also be assessed against any SAR/visible imagery if available.
